src/components/ai-input.tsx

```diff
@@ -131,10 +135,51 @@ const UnforwardedAIInput = (
 		}, 400 );
 	};
 
+	useEffect( () => {
+		if ( isAssistantThinking ) {
+			thinkingTimeout.current.push(
+				setTimeout( () => {
+					setThinkingDuration( 'medium' );
+				}, 3000 )
+			);
+
+			thinkingTimeout.current.push(
+				setTimeout( () => {
+					setThinkingDuration( 'long' );
+				}, 5000 )
```

Comment:
What do you think if we expand the time between these.


```suggestion
				}, 7000 )
```